src/data/pfunctor/univariate/M.lean

```diff
@@ -277,7 +277,7 @@ begin
   dsimp only [M.mk,dest],
   cases x with x ch, congr' with i,
   cases h : ch i,
-  simp  only [children,M.approx.s_mk,children',cast_eq],
+  simp only [children,M.approx.s_mk,children',cast_eq],
```

Comment:
This one turns out to have other (preexisting) whitespace errors not caught by your filter, do we care?
```
  simp only [children, M.approx.s_mk, children', cast_eq],
```